### PR TITLE
Modify to support to rebuild multi indexes

### PR DESCRIPTION
**Pull request by laura-ding**

### CODE REVIEW

File: src/storage/test/RebuildIndexTest.cpp [hunk 1]
```diff
@@ -75,7 +75,8 @@ TEST_F(RebuildIndexTest, RebuildTagIndexCheckALLData) {
     parameter.set_space_id(1);
     std::vector<PartitionID> parts = {1, 2, 3, 4, 5, 6};
     parameter.set_parts(parts);
-    parameter.task_specfic_paras.emplace_back("4,5");
+    parameter.task_specfic_paras.emplace_back("4");
```
Comment (darionyaphet) [inline src/storage/test/RebuildIndexTest.cpp:78]: IF we will add more argument about rebuild how to determine the index id bounds ？

Comment (laura-ding) [reply src/storage/test/RebuildIndexTest.cpp:78]: Can you tell me what parameters will be supported?  I think every parameter in the vector means the same meaning. so the user  use it is easy. and it no need to tell user the firstly parameter mean what, the second mean what.

**Comment by critical27** [comment]
Better to wait #292 merge at first.

File: src/meta/processors/jobMan/ListEdgeIndexStatusProcessor.cpp [hunk 2]
```diff
@@ -52,7 +52,13 @@ void ListEdgeIndexStatusProcessor::process(const cpp2::ListIndexStatusReq& req)
...
     std::unordered_map<std::string, cpp2::JobStatus> tmp;
     for (auto &jobDesc : jobs) {
-        tmp.emplace(jobDesc.get_paras()[0], jobDesc.get_status());
+        auto paras = jobDesc.get_paras();
+        if (paras.size() == 1) {
+            tmp.emplace(paras[0] + "_all_edge_indexes", jobDesc.get_status());
+            continue;
+        }
+        paras.pop_back();
+        tmp.emplace(folly::join(",", paras), jobDesc.get_status());
```
Comment (bright-starry-sky) [inline src/meta/processors/jobMan/ListEdgeIndexStatusProcessor.cpp:61]: In graph layer，I saw that null argument is allowed, meaning that all indexes. How to unify with the storage layer?
https://github.com/vesoft-inc/nebula-graph/pull/600，

Comment (laura-ding) [reply src/meta/processors/jobMan/ListEdgeIndexStatusProcessor.cpp:61]: The same goes for Storage.You can see the RebuildJobExecutor.cpp

Comment (bright-starry-sky) [reply src/meta/processors/jobMan/ListEdgeIndexStatusProcessor.cpp:61]: > The same goes for Storage.You can see the RebuildJobExecutor.cpp

Got, We do that in code `RebuildIndexTask.cpp`. nice job.

**Review by bright-starry-sky — APPROVED** [review]
LGTM

File: src/storage/test/RebuildIndexTest.cpp (hunk 1 above)
Comment (darionyaphet) [reply src/storage/test/RebuildIndexTest.cpp:78]: There are hidden dangers here

**Review by darionyaphet — APPROVED** [review]

**Pull request merged**